uefi/src/main.rs

```diff
@@ -63,39 +69,77 @@ fn efi_main(image: Handle, st: SystemTable<Boot>) -> Status {
 
 fn main_inner(image: Handle, mut st: SystemTable<Boot>) -> Status {
     // temporarily clone the y table for printing panics
+
     unsafe {
         *SYSTEM_TABLE.get() = Some(st.unsafe_clone());
     }
+    let stdout = unsafe { &mut *SYSTEM_TABLE.get() };
+    let stdout = stdout.as_mut().unwrap();
```

Comment:
The only reason for the `SYSTEM_TABLE` static is that we can print on panics. We can use `st` directly here.

```suggestion
```